index.html

```diff
@@ -30,6 +30,247 @@ <h3 class="supporting-text">I'am a Software Developer</h3>
+                        </div>
+                    </div>
+                </div>
+                                <!--About Me Section-->
+                <div class="about">
+                    <h1>About me</h1>
+                    <p class="description">
+                        Hello I'm a software developer! I can help
+                        you build a product , feature or website 
+                        Look through some of my work and experience! 
+                        If you like what you see and have a project you need coded,
+                        don't hestiate to contact me.
+                    </p>
+                    <button type="submit">Get My Resume</button>
+                </div>
+            </div>
+            <div class="skills">
+                <hr class="about-hr">
+                <div class="languages">
+                   <div class="box">
+                    <img src="./images/Rectangle55.png" alt="">
+                    <h1 class="head1">Language</h1>
+                   </div>
+                    <ul>
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>Css</li>
+                    </ul>
+                </div>
+                <div class="frameworks">
+                    <div class="box2">
+                        <img src="./images/Rectangle56.png" alt="">
+                        <h1 class="head2">Frameworks</h1>
+                    </div>
+                    <ul>
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RSPec</li>
+                        <li>CapyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+                <div class="skills-list">
+                    <div class="box3">
+                        <img src="./images/Rectangle57.png" alt="">
+                        <h1 class="head3">Skills</h1>
+                    </div>
+                    <ul>
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="contact-form">
+                <p class="contact-form-text">
+                    I'm always interested in hearing about new projects,
+                    so if you'd like to chat please get in touch.
+                </p>
+                <form class="text-form">
+                    <input type="text" placeholder="Full Name">
+                    <input type="email" placeholder="Email Address">
+                    <textarea cols="30" rows="10">Enter text here</textarea>
+                    <button type="submit" class="contact-form-button">Get in touch</button>
```

Comment:
- [ ] good job on the form so far but please try to make it as similar as possible to the figma template. see screenshots below

your form
![Screenshot from 2022-07-01 05-44-25](https://user-images.githubusercontent.com/66230306/176945588-02cb5a58-9f1e-4db4-be36-8a506b991dc0.png)

figma template form
![Screenshot from 2022-07-01 05-45-04](https://user-images.githubusercontent.com/66230306/176945672-6cbce6a1-78fb-49f6-a594-4485e15a8112.png)